packages/client/mutations/handlers/handleGitLabCreateIssue.ts

```diff
@@ -0,0 +1,55 @@
+import {ConnectionHandler, RecordProxy, RecordSourceSelectorProxy} from 'relay-runtime'
+import createProxyRecord from '~/utils/relay/createProxyRecord'
+import toTeamMemberId from '../../utils/relay/toTeamMemberId'
+import {CreateTaskMutationResponse} from '../../__generated__/CreateTaskMutation.graphql'
+import getGitLabProjectsConn from '../connections/getGitLabProjectsConn'
+import {parseWebPath} from './../../utils/parseWebPath'
+
+const handleGitLabCreateIssue = (
+  task: RecordProxy<NonNullable<CreateTaskMutationResponse['createTask']['task']>>,
+  store: RecordSourceSelectorProxy
+) => {
+  const integration = task.getLinkedRecord('integration')
+  if (!integration) return
+  const webPath = integration.getValue('webPath') as string | undefined
+  const {fullPath} = webPath ? parseWebPath(webPath) : {fullPath: ''}
+  const project = createProxyRecord(store, '_xGitLabProject', {
+    fullPath
+  })
+  const issueEdge = createProxyRecord(store, '_xGitLabIssueEdge', {})
+  issueEdge.setLinkedRecord(integration, 'node')
+
+  const issueConn = createProxyRecord(store, '_xGitLabIssueConnection', {})
+  issueConn.setLinkedRecords([issueEdge], 'edges')
+
+  const issueArgs = {first: 25, includeSubepics: true, sort: 'UPDATED_DESC', state: 'opened'}
```

Comment:
-1 single source of truth required here. if i change these vars in the gql request, this whole thing will break.

Reply:
Makes sense. I'm now passing the variables into the `GitLabScopingSearchResultsRoot` query here: https://github.com/ParabolInc/parabol/blob/fed8dce61509d31113972d785915730d520224c0/packages/client/components/GitLabScopingSearchResultsRoot.tsx#L43